examples/async_printing.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
```

Comment:
Nice example.  Consider making use of set_window_title() somewhere.

Reply:
Added set_window_title() to example